content/docs/started/getting-started-k8s.md

```diff
@@ -4,84 +4,156 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
 weight = 4
 +++
 
-Follow these instructions if you want to install Kubeflow on an existing Kubernetes
-cluster.
+Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-If you are using a Kubernetes distribution or Cloud Provider which has specific
-instructions for installing Kubeflow we recommend following those instructions.
-Those instructions do additional Cloud specific setup to create a really great 
-Kubeflow experience.
+If you are using a Kubernetes distribution or Cloud Provider which has specific instructions for installing Kubeflow we recommend following those instructions. Those instructions do additional Cloud specific setup to create a really great Kubeflow experience.
 
-You can use either [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage) or
-[Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to install a Kubernetes cluster and it
-is recommended to use [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage)
-as it is more simple to install.
+For installing Kubeflow on an existing Kubernetes Cluster, you can use one of the following options:
```

Comment:
This paragraph combined with the single entry in the list below implies that there's only one option for deploying Kubeflow to existing clusters, and that that option is maintained by Arrikto.

This content was not in the original PR https://github.com/kubeflow/website/pull/833/files that I reviewed a while ago, and I'm wondering if I've missed some conversations.

I'm concerned about saying that this particular solution is maintained by a third party rather than by the Kubeflow community.

It's also not clear why this particular solution is special and needs to be maintained by a particular group. This document should describe the recommended solution, without giving people cause to think that the solution is possibly a special case rather than the recommended one.

@jlewi and @abhi-g and @yanniszark do you have any input about the reason for adding the special mention of Arrikto into the docs?

Reply:
Indeed, the changes in the PR are a result of the conversations in https://github.com/kubeflow/kubeflow/pull/3538.

To sum up:

After discussing with @jlewi we agreed on the following:
Existing, as a name, is too general and there can be multiple ways for deploying on existing clusters.
Some users might want something different from what we currently do (eg to bring their own Istio).
In addition, we agreed that since there isn't 1 correct way to deploy on existing clusters, each solution should have an arbitrating party, to clearly communicate whose opinions are represented.

> This paragraph combined with the single entry in the list below implies that there's only one option for deploying Kubeflow to existing clusters, and that that option is maintained by Arrikto.

To the best of my knowledge, at this particular time, it is the only config file specifically tailored for deploying on an existing Kubernetes Cluster and also the only one that supports authentication.

> I'm concerned about saying that this particular solution is maintained by a third party rather than by the Kubeflow community.

[From our discussion with @jlewi](https://github.com/kubeflow/kubeflow/pull/3538#issuecomment-506771292), I understand that the current platforms (GCP, AWS) are arbitrated by Google and Amazon respectively, not the community.
While the whole community contributes to them, ultimately there is an arbitrating group who decides the contents of the config file.

That is the case as well with `existing_arrikto`.
Since there are many possible solutions and we are following a particular one, this config represents our opinion and as @jlewi suggested, that should be clear in the name and documentation.

> It's also not clear why this particular solution is special and needs to be maintained by a particular group.

Because there is not 1 universal solution, this solution represents a reference architecture targeted at existing clusters and providing auth functionality that wasn't available for existing clusters yet.
Since there are many solutions, we have to make some assumptions. These assumptions and opinions are why this platform is arbitrated by Arrikto.
As @jlewi put it: 

> Pick a name e.g. "arrikto" that clearly conveys whose oppinions are represented and who is the arbitrator (and message it as such)

With that said, anyone from the community can contribute and there have already been contributions from outside Arrikto (eg https://github.com/kubeflow/kubeflow/pull/3607)

cc @jlewi

Reply:
Thank you for the clarifications! This LGTM.